solutions/standard/README.md

```diff
@@ -0,0 +1,94 @@
+# VSI on VPC IBM Secure Landing Zone - Standard Variation
```

Comment:
Please see all of the comments that I made against the README for the quickstart variation and address here as well.  I believe most of them apply to standard variation too.

Reply:
Made changes to Quickstart & Standard variation